scripts/apply_weights.py

```diff
@@ -136,9 +135,9 @@
 
 # Add information (hypermutated samples and if they were used to train model)
 final_pred = final_pred.assign(hypermutated=1)
-final_pred.loc[final_pred['Silent per Mb'] < 2 *
-               final_pred['Silent per Mb'].std(), 'hypermutated'] = 0
+final_pred.loc[final_pred['log10_mut'] < 5 * final_pred['log10_mut'].std(),
```

Comment:
i think i will leave this hardcoded in to ensure consistency. The goal of this line is to remove the hypermutated outliers if the user thinks that adjusting for them in the model is not sufficient to control for their effect. 5 std ends up removing about the top 300 samples if specified